pkg/dashboards/ifaces.go

```diff
@@ -6,6 +6,8 @@ import "github.com/grafana/grafana/pkg/models"
 type Store interface {
 	// ValidateDashboardBeforeSave validates a dashboard before save.
 	ValidateDashboardBeforeSave(dashboard *models.Dashboard, overwrite bool) (bool, error)
+	// GetFolderByTitle retrieves a dashboard by its title and it's used by unified alerting
```

Comment:
Nit, but I think the following reads better:
```suggestion
	// GetFolderByTitle retrieves a dashboard by its title and is used by unified alerting
```